### PR TITLE
feat(css): add roboto font files

### PR DESCRIPTION
**Pull request by manucorporat**
#### Changes proposed in this pull request:

-
-
-

**Ionic Version**: 1.x / 2.x / 3.x / 4.x

**Fixes**: #

### CODE REVIEW

**Comment by mburger81** [comment]
are you sure you need only
```
font-style: normal;
font-weight: 400;
```
I think I saw also font-weight: 500 used in your components.

**Comment by brandyscarney** [comment]
@manucorporat Do we want to get this in for 4.1?

File: core/src/css/font-roboto.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* stylelint-disable */
```
Comment (brandyscarney) [inline core/src/css/font-roboto.scss]: This isn't needed with the file being in the stylelintrc

```suggestion

```

**Comment by brandyscarney** [comment]
Closing in favor of https://github.com/ionic-team/ionic/pull/18934, discussed this with Manu! 

I will add you as a co-author @mburger81. Thank you!

**Pull request closed without merging**